### PR TITLE
add mixed precision doc

### PR DESCRIPTION
**Pull request by danleifeng**
add mixed precision doc

### CODE REVIEW

File: docs/mixed_precision.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## 简介
+PLSC支持混合精度训练。使用混合精度训练可以提升训练的速度，同时减少训练使用的显存开销。
+
+## 使用方法
+可以通过下面的代码设置开启混合精度训练：
+
+```python
+from __future__ import print_function
+import plsc.entry as entry
```
Comment (sandyhouse) [inline docs/mixed_precision.md]: Now we can import Entry from plsc directly, e.g., from plsc import Entry

File: docs/mixed_precision.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+def main():
+    ins = entry.Entry()
```
Comment (sandyhouse) [inline docs/mixed_precision.md]: This line can be just:
```python
ins = Entry()
```

File: docs/mixed_precision.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- `loss_scaling`：float类型，为初始的损失缩放值，这个值有可能会影响混合精度训练的精度，建议设为默认值1.0。
+
+为了提高混合精度训练的稳定性和精度，默认开启了动态损失缩放机制。更多关于混合精度训练的介绍可参考：[混合精度训练](https://arxiv.org/abs/1710.03740)
+
+## 训练性能
+
+| 模型\速度(单机8卡) | 正常训练 | 混合精度训练 | 加速比 |
+| --- | --- | --- | --- |
+| ResNet50 | 2567.96 images/s | 3643.11 images/s | 1.42 |
+备注：上述模型训练使用的loss_type均为'dist_arcface'。
```
Comment (sandyhouse) [inline docs/mixed_precision.md]: Add hardware configurations please.

File: docs/mixed_precision.md (hunk 1 above)
Comment (danleifeng) [reply docs/mixed_precision.md]: done.thanks!

File: docs/mixed_precision.md (hunk 2 above)
Comment (danleifeng) [reply docs/mixed_precision.md]: done. thanks!

File: docs/mixed_precision.md (hunk 3 above)
Comment (danleifeng) [reply docs/mixed_precision.md]: done,thansk!

**Pull request merged**